newIDE/app/src/UI/PriceTag.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import * as React from 'react';
+import { I18n } from '@lingui/react';
+
+import Text from './Text';
+import { makeStyles } from '@material-ui/core';
+
+type Props = {|
+  value: number,
+  /**
+   * To be used when the component is over an element for which
+   * we don't control the background (e.g. an image).
+   * This component should logically be invariant to selected theme
+   * when this option is activated
```

Comment:
Not sure I totally understood the comment

Reply:
I'll clarify